perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -500,9 +500,11 @@ def _parse_arguments(argv):
   )
 
   parser.add_argument(
-      'spreadsheet_id',
+      '--spreadsheet_id',
       help='Provide id of spreadsheet',
       action='store',
+      default="",
```

Comment:
Do you need checks for empty spreadsheet_id in the code? Did you run the list benchmarks too? Please update the PR description accordingly